code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -700,18 +700,20 @@
 						if(do_after(usr,usr,30))
 							visible_message("<span class = 'warning'><b>[name]</b> unleashes a [pick("tremendous","gigantic","colossal")] fart!</span>","<span class = 'warning'>You hear a [pick("tremendous","gigantic","colossal")] fart.</span>")
 							playsound(location, 'sound/effects/superfart.ogg', 50, 0)
-							if(!wearing_suit)
-								for(var/mob/living/V in view(src,aoe_range))
-									if(!airborne_can_reach(location,get_turf(V),aoe_range))
-										continue
-									shake_camera(V,10,5)
-									if (V == src)
-										continue
-									to_chat(V, "<span class = 'danger'>You're sent flying!</span>")
-									V.Knockdown(5) // why the hell was this set to 12 christ
-									step_away(V,location,15)
-									step_away(V,location,15)
-									step_away(V,location,15)
+							for(var/mob/living/V in view(src,aoe_range))
+								if(!airborne_can_reach(location,get_turf(V),aoe_range))
+									continue
+								shake_camera(V,10,5)
+								if (V == src)
+									continue
+								to_chat(V, "<span class = 'danger'>You're sent flying!</span>")
+								V.Knockdown(5) // why the hell was this set to 12 christ
+								step_away(V,location,15)
+								step_away(V,location,15)
+								step_away(V,location,15)
+							if !turf.has_gravity(mob)
```

Comment:
Also I don't think `mob` and `turf` are defined here. I used those to let you fill them in because I was on my phone and couldn't be bothered to type it out fully.